### PR TITLE
Component getSchools Front-end

### PR DESCRIPTION
**Pull request by okozmovskaya**
Guys, please, check my PR. The component getSchool is not complete. We should add it filter function, link for read/ save comments, save rating and save school, but now we can use it with all our schools.
In package-json (main-folder) I installed dev-dependency "Concurrently" which to allow run server and app with one "npm run dev". Look at the script in package-json (main-folder). You can try to use it or delete my changes in package-json (main-folder) from PR.

This PR connect with issue #6 and #5

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,4 +1,4 @@
-MONGOBD_URI= mongodb+srv://Ira:0sZATpGxXHT6EOgs@cluster0.l1fuv.mongodb.net/hyf?retryWrites=true&w=majority
+MONGOBD_URI = mongodb+srv://Olga:OEKaEKn5hG8haB8M@cluster0.l1fuv.mongodb.net/hyf?retryWrites=true&w=majority
```
Comment (Feruzteame) [inline .env:1]: I think you do not need to push your private DB connect.

**Review by Feruzteame — APPROVED** [review]

File: .env (hunk 1 above)
Comment (okozmovskaya) [reply .env:1]: Yes, you are right. I added file .env in both .gitignore, but this doesn't work. I don't know how we can not join some files or folders another way.... May be you know?

**Comment by IrynaSpyrydonova** [comment]
@okozmovskaya looks very nice, good job👍 Just one thing,  there is already existing router to get all schools (app.get('/schools) in index.js, so just not be confuse which one to use - please delete it, otherwise we have 2 the same routers.

**Review by IrynaSpyrydonova — APPROVED** [review]

**Comment by okozmovskaya** [comment]
@Feruzteame , @IrynaSpyrydonova thank you for your approve)) I deleted the file from route-folder with the same router. 
Also created the star-raiting with correct css and add the component for Search-bar.
@fmkarakus if it's ok for your, please merge it.

The working on "send filter-data to sibling component" is continuing ...

**Review by fmkarakus — APPROVED** [review]

**Pull request merged**